java/RSPEC-2178.java

```diff
@@ -0,0 +1,8 @@
+class BinaryInsteadOfBoolean {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout comment separator -->
---
RSPEC-2178: Potential bug: bitwise operator inside if condition
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._